[2025-01-04T21:49:23.403+0000] {processor.py:161} INFO - Started process (PID=418) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:49:23.404+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:49:23.405+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:49:23.405+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:49:23.409+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:49:23.408+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:49:23.409+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:49:23.423+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.025 seconds
[2025-01-04T21:49:53.680+0000] {processor.py:161} INFO - Started process (PID=428) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:49:53.681+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:49:53.683+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:49:53.682+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:49:53.687+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:49:53.686+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:49:53.687+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:49:53.706+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.032 seconds
[2025-01-04T21:50:23.974+0000] {processor.py:161} INFO - Started process (PID=438) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:50:23.976+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:50:23.977+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:50:23.977+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:50:23.982+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:50:23.981+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:50:23.982+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:50:23.999+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T21:50:54.262+0000] {processor.py:161} INFO - Started process (PID=448) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:50:54.263+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:50:54.266+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:50:54.266+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:50:54.270+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:50:54.269+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:50:54.270+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:50:54.287+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T21:51:24.556+0000] {processor.py:161} INFO - Started process (PID=458) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:51:24.557+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:51:24.559+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:51:24.559+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:51:24.563+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:51:24.562+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:51:24.564+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:51:24.581+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T21:51:54.838+0000] {processor.py:161} INFO - Started process (PID=468) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:51:54.838+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:51:54.840+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:51:54.840+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:51:54.844+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:51:54.843+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:51:54.844+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:51:54.860+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T21:52:25.122+0000] {processor.py:161} INFO - Started process (PID=478) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:52:25.123+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:52:25.125+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:52:25.125+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:52:25.130+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:52:25.129+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:52:25.130+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:52:25.146+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T21:52:55.398+0000] {processor.py:161} INFO - Started process (PID=488) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:52:55.399+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:52:55.401+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:52:55.400+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:52:55.405+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:52:55.404+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:52:55.406+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:52:55.423+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.032 seconds
[2025-01-04T21:53:25.683+0000] {processor.py:161} INFO - Started process (PID=498) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:53:25.684+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:53:25.685+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:53:25.685+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:53:25.689+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:53:25.688+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:53:25.690+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:53:25.707+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.030 seconds
[2025-01-04T21:53:55.970+0000] {processor.py:161} INFO - Started process (PID=508) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:53:55.971+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:53:55.973+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:53:55.972+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:53:55.979+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:53:55.978+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:53:55.980+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:53:55.995+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T21:54:26.297+0000] {processor.py:161} INFO - Started process (PID=518) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:54:26.298+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:54:26.301+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:54:26.300+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:54:26.306+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:54:26.304+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:54:26.306+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:54:26.320+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.029 seconds
[2025-01-04T21:54:56.602+0000] {processor.py:161} INFO - Started process (PID=528) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:54:56.603+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:54:56.605+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:54:56.605+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:54:56.610+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:54:56.609+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:54:56.611+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:54:56.627+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.032 seconds
[2025-01-04T21:55:26.968+0000] {processor.py:161} INFO - Started process (PID=538) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:55:26.970+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:55:26.972+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:55:26.972+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:55:26.977+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:55:26.976+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:55:26.978+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:55:26.994+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T21:55:57.353+0000] {processor.py:161} INFO - Started process (PID=548) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:55:57.354+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:55:57.356+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:55:57.356+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:55:57.360+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:55:57.359+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:55:57.361+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:55:57.379+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.032 seconds
[2025-01-04T21:56:27.717+0000] {processor.py:161} INFO - Started process (PID=558) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:56:27.719+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:56:27.721+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:56:27.721+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:56:27.725+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:56:27.724+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:56:27.726+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:56:27.745+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.034 seconds
[2025-01-04T21:56:58.036+0000] {processor.py:161} INFO - Started process (PID=568) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:56:58.037+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:56:58.038+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:56:58.038+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:56:58.043+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:56:58.042+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:56:58.043+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:56:58.060+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T21:57:28.346+0000] {processor.py:161} INFO - Started process (PID=578) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:57:28.347+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:57:28.348+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:57:28.348+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:57:28.351+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:57:28.350+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:57:28.351+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:57:28.365+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.023 seconds
[2025-01-04T21:57:58.666+0000] {processor.py:161} INFO - Started process (PID=588) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:57:58.667+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:57:58.669+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:57:58.668+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:57:58.672+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:57:58.671+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:57:58.673+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:57:58.688+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.027 seconds
[2025-01-04T21:58:29.015+0000] {processor.py:161} INFO - Started process (PID=598) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:58:29.016+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:58:29.018+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:58:29.017+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:58:29.021+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:58:29.021+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:58:29.022+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:58:29.041+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T21:58:59.353+0000] {processor.py:161} INFO - Started process (PID=608) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:58:59.354+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:58:59.356+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:58:59.356+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:58:59.360+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:58:59.359+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:58:59.360+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:58:59.375+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.027 seconds
[2025-01-04T21:59:29.706+0000] {processor.py:161} INFO - Started process (PID=618) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:59:29.707+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T21:59:29.709+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:59:29.708+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:59:29.713+0000] {logging_mixin.py:188} INFO - [2025-01-04T21:59:29.711+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T21:59:29.713+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T21:59:29.730+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.030 seconds
[2025-01-04T22:00:00.030+0000] {processor.py:161} INFO - Started process (PID=628) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:00:00.031+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:00:00.033+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:00:00.032+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:00:00.037+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:00:00.036+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:00:00.037+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:00:00.053+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.030 seconds
[2025-01-04T22:00:30.340+0000] {processor.py:161} INFO - Started process (PID=638) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:00:30.341+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:00:30.343+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:00:30.342+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:00:30.346+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:00:30.346+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:00:30.347+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:00:30.363+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T22:01:00.668+0000] {processor.py:161} INFO - Started process (PID=648) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:01:00.669+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:01:00.671+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:01:00.670+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:01:00.674+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:01:00.673+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:01:00.675+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:01:00.690+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T22:01:30.952+0000] {processor.py:161} INFO - Started process (PID=658) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:01:30.953+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:01:30.954+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:01:30.954+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:01:30.958+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:01:30.957+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:01:30.958+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:01:30.973+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.027 seconds
[2025-01-04T22:02:01.273+0000] {processor.py:161} INFO - Started process (PID=668) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:02:01.274+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:02:01.276+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:02:01.276+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:02:01.279+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:02:01.279+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:02:01.280+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:02:01.295+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.026 seconds
[2025-01-04T22:02:31.563+0000] {processor.py:161} INFO - Started process (PID=678) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:02:31.563+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:02:31.565+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:02:31.565+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:02:31.568+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:02:31.567+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:02:31.569+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:02:31.584+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.027 seconds
[2025-01-04T22:03:01.909+0000] {processor.py:161} INFO - Started process (PID=688) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:03:01.910+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:03:01.912+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:03:01.911+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:03:01.915+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:03:01.915+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:03:01.916+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:03:01.935+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T22:03:32.227+0000] {processor.py:161} INFO - Started process (PID=698) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:03:32.228+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:03:32.230+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:03:32.229+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:03:32.233+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:03:32.232+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:03:32.234+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:03:32.250+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T22:04:02.550+0000] {processor.py:161} INFO - Started process (PID=708) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:04:02.551+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:04:02.553+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:04:02.552+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:04:02.556+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:04:02.555+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:04:02.557+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:04:02.573+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.027 seconds
[2025-01-04T22:04:32.912+0000] {processor.py:161} INFO - Started process (PID=718) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:04:32.912+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:04:32.914+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:04:32.913+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:04:32.917+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:04:32.916+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:04:32.917+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:04:32.933+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.025 seconds
[2025-01-04T22:05:03.251+0000] {processor.py:161} INFO - Started process (PID=728) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:05:03.253+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:05:03.255+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:05:03.255+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:05:03.260+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:05:03.259+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:05:03.261+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:05:03.280+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.035 seconds
[2025-01-04T22:05:33.588+0000] {processor.py:161} INFO - Started process (PID=738) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:05:33.589+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:05:33.591+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:05:33.591+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:05:33.595+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:05:33.594+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:05:33.596+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:05:33.612+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.029 seconds
[2025-01-04T22:06:03.926+0000] {processor.py:161} INFO - Started process (PID=748) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:06:03.927+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:06:03.930+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:06:03.929+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:06:03.934+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:06:03.933+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:06:03.934+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:06:03.950+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.029 seconds
[2025-01-04T22:06:34.253+0000] {processor.py:161} INFO - Started process (PID=758) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:06:34.254+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:06:34.256+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:06:34.255+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:06:34.259+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:06:34.258+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:06:34.260+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:06:34.275+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T22:07:04.576+0000] {processor.py:161} INFO - Started process (PID=768) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:07:04.577+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:07:04.580+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:07:04.580+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:07:04.584+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:07:04.583+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:07:04.584+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:07:04.601+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T22:07:34.931+0000] {processor.py:161} INFO - Started process (PID=778) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:07:34.932+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:07:34.933+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:07:34.933+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:07:34.938+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:07:34.937+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:07:34.938+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:07:34.955+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.031 seconds
[2025-01-04T22:08:05.231+0000] {processor.py:161} INFO - Started process (PID=788) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:08:05.232+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:08:05.234+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:08:05.234+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:08:05.239+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:08:05.238+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:08:05.239+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:08:05.260+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.035 seconds
[2025-01-04T22:08:35.499+0000] {processor.py:161} INFO - Started process (PID=798) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:08:35.500+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:08:35.501+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:08:35.501+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:08:35.505+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:08:35.504+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:08:35.505+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:08:35.522+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.029 seconds
[2025-01-04T22:09:05.788+0000] {processor.py:161} INFO - Started process (PID=808) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:09:05.788+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:09:05.789+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:09:05.789+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:09:05.792+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:09:05.791+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:09:05.792+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:09:05.803+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.019 seconds
[2025-01-04T22:09:36.100+0000] {processor.py:161} INFO - Started process (PID=818) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:09:36.100+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:09:36.102+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:09:36.102+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:09:36.105+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:09:36.104+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:09:36.106+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:09:36.121+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.026 seconds
[2025-01-04T22:10:06.392+0000] {processor.py:161} INFO - Started process (PID=828) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:10:06.393+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:10:06.395+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:10:06.394+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:10:06.399+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:10:06.398+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:10:06.399+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:10:06.415+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T22:10:36.704+0000] {processor.py:161} INFO - Started process (PID=838) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:10:36.705+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:10:36.706+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:10:36.706+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:10:36.710+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:10:36.709+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:10:36.710+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:10:36.726+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T22:11:07.023+0000] {processor.py:161} INFO - Started process (PID=848) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:11:07.024+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:11:07.025+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:11:07.025+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:11:07.029+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:11:07.028+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 8
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:11:07.029+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:11:07.046+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T22:11:34.323+0000] {processor.py:161} INFO - Started process (PID=858) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:11:34.324+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:11:34.325+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:11:34.325+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:11:34.328+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:11:34.327+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 25
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:11:34.328+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:11:34.342+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.022 seconds
[2025-01-04T22:12:04.692+0000] {processor.py:161} INFO - Started process (PID=868) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:12:04.693+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:12:04.695+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:12:04.694+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:12:04.698+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:12:04.697+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 25
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:12:04.699+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:12:04.716+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.029 seconds
[2025-01-04T22:12:35.164+0000] {processor.py:161} INFO - Started process (PID=878) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:12:35.165+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:12:35.166+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:12:35.166+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:12:35.170+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:12:35.169+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 25
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:12:35.170+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:12:35.185+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.027 seconds
[2025-01-04T22:13:05.518+0000] {processor.py:161} INFO - Started process (PID=888) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:13:05.519+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:13:05.520+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:13:05.520+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:13:05.524+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:13:05.523+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 25
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:13:05.525+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:13:05.541+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.030 seconds
[2025-01-04T22:13:35.932+0000] {processor.py:161} INFO - Started process (PID=898) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:13:35.933+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T22:13:35.934+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:13:35.934+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:13:35.938+0000] {logging_mixin.py:188} INFO - [2025-01-04T22:13:35.937+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 25
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T22:13:35.939+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T22:13:35.955+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.028 seconds
[2025-01-04T23:53:06.745+0000] {processor.py:161} INFO - Started process (PID=903) to work on /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T23:53:06.748+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/raw_banking_tinkoff.py for tasks to queue
[2025-01-04T23:53:06.750+0000] {logging_mixin.py:188} INFO - [2025-01-04T23:53:06.749+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T23:53:06.758+0000] {logging_mixin.py:188} INFO - [2025-01-04T23:53:06.758+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/raw_banking_tinkoff.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/raw_banking_tinkoff.py", line 25
    1=1
    ^
SyntaxError: cannot assign to literal here. Maybe you meant '==' instead of '='?
[2025-01-04T23:53:06.760+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/raw_banking_tinkoff.py
[2025-01-04T23:53:06.776+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/raw_banking_tinkoff.py took 0.036 seconds
